src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -270,3 +270,8 @@ case class MesosFrameworkMessageEvent(
     message: Array[Byte],
     eventType: String = "framework_message_event",
     timestamp: String = Timestamp.now().toString) extends MarathonEvent
+
+case class ReconciliationStatusUpdate(
```

Comment:
You mean to

`InstanceTracker.process` ->
`InstanceTrackerDelegate.process` ->
`InstanceUpdateActor.Receive(UpdateContext)` ->
`InstanceUpdateActor.processUpdate` ->
`InstanceUpdateOpResolver.resolve` ->
`InstanceUpdater.mesosUpdate` ->
`InstanceChangedEventsGenerator.events` ->
`new Events.InstanceChanged` ?

None really, I just didn't know it could be used for this 😆

Reply:
I don't quite understand the flow. As a consumer of the events you should not really care how the event was generated.

Reply:
I wasn't aware of the `InstanceChanged` event. Now I know it, but I was surprised to see the chain that I had to follow in order to find it :D